cmd/controller-manager/main.go

```diff
@@ -62,16 +62,17 @@ func main() {
 	if err != nil {
 		log.Fatalf("Error loading logging configuration: %v", err)
 	}
-	config, err := logging.NewConfigFromMap(cm, logconfig.ControllerManager)
+	cfg, err := logging.NewConfigFromMap(cm, logconfig.ControllerManager)
 	if err != nil {
 		log.Fatalf("Error parsing logging configuration: %v", err)
 	}
-	logger, atomicLevel := logging.NewLoggerFromConfig(config, logconfig.ControllerManager)
+	logger, atomicLevel := logging.NewLoggerFromConfig(cfg, logconfig.ControllerManager)
```

Comment:
Do we use `atomicLevel` anywhere? If not, should we name it "_"?

Reply:
it is used in the watch call